samples/generated_samples/logging_v2_generated_config_service_v2_copy_log_entries_async.py

```diff
@@ -36,10 +36,10 @@
 
 async def sample_copy_log_entries():
     # Create a client
-    client = logging_v2.ConfigServiceV2AsyncClient()
+    client = logging_v2.services.config_service_v2.ConfigServiceV2AsyncClient()
 
     # Initialize request argument(s)
-    request = logging_v2.CopyLogEntriesRequest(
+    request = logging_v2.types.CopyLogEntriesRequest(
```

Comment:
One thought: these generated samples aren't covered by tests. Have you considered adding tests for these to catch this kind of issue in the future, like the [other snippets](https://github.com/googleapis/python-logging/tree/c201482c913b5f2eac82359787511a5a92fc3406/samples/snippets)? Smoke tests could be helpful here (i.e. just executing each function and making sure nothing crashes)

There may be a simple way to programmatically load and execute each sample, but if it's bit more complicated we could open a bug for later